pyproject.toml

```
@@ -22,10 +22,10 @@ dependencies = [
     'photutils >=1.10.0',
     'pyparsing >=2.4.7',
     'requests >=2.26',
-    'rad >= 0.18.0',
-    # 'rad @ git+https://github.com/spacetelescope/rad.git@main',
-    'roman_datamodels >= 0.18.0',
-    # 'roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git@main',
+    # 'rad >= 0.18.0',
+    'rad @ git+https://github.com/spacetelescope/rad.git@main',
+    # 'roman_datamodels >= 0.18.0',
+    'roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git@main',
```

```suggestion
    'rad @ git+https://github.com/spacetelescope/rad.git',
    # 'roman_datamodels >= 0.18.0',
    'roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git',
```
Can you try making this change?